bentoml/_internal/frameworks/pickle.py

```diff
@@ -0,0 +1,281 @@
+            BentoML modelstore, provided by DI Container.
+
+    Returns:
+        :obj: an instance of :obj: model from BentoML modelstore.
+
+    Examples:
+
+    .. code-block:: python
+
+        import bentoml
+
+        unpickled_model = bentoml.pickle.load('my_model:latest')
+    """  # noqa
+    _, model_file = _get_model_info(tag, model_store)
+    with open(model_file, "rb") as f:
+        # The protocol version used is detected automatically, so we do not
+        # have to specify it.
+        return pickle.load(f)
+
+
+@inject
+def save(
+    name: str,
+    model: t.Any,
+    *,
+    metadata: t.Optional[t.Dict[str, t.Any]] = None,
+    model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
+) -> Tag:
+    """
+    Save a model instance to BentoML modelstore.
+
+    Args:
+        name (:code:`str`):
+            Name for given model instance. This should pass Python identifier check.
+        model:
+            Instance of model to be saved.
+        metadata (:code:`Dict[str, Any]`, `optional`,  default to :code:`None`):
+            Custom metadata for given model.
+        model_store (:mod:`~bentoml._internal.models.store.ModelStore`, default to :mod:`BentoMLContainer.model_store`):
+            BentoML modelstore, provided by DI Container.
+
+    Returns:
+        :obj:`~bentoml._internal.types.Tag`: A :obj:`tag` with a format `name:version` where `name` is the user-defined model's name, and a generated `version` by BentoML.
+
+    Examples:
+
+    .. code-block:: python
+
+        import bentoml
+
+        class MyCoolModel:
+            def predict(self, some_integer: int):
+                return some_integer**2
+
+        model_to_save = MyCoolModel();
+        tag_info = bentoml.pickle.save("test_pickle_model", model_to_save)
+        runner = bentoml.pickle.load_runner(tag_info)
+        runner.run(3)
+
+    """  # noqa
+    context = {"framework_name": "pickle"}
+
+    _model = Model.create(
+        name,
+        module=MODULE_NAME,
+        metadata=metadata,
+        context=context,
+    )
+
+    with open(_model.path_of(f"{SAVE_NAMESPACE}{PKL_EXT}"), "wb") as f:
+        cloudpickle.dump(model, f)
+
+    _model.save(model_store)
+    return _model.tag
+
+
+class _PickleRunner(Runner):
+    @inject
+    def __init__(
+        self,
+        tag: Tag,
+        function_name: str,
+        name: str,
+        resource_quota: t.Optional[t.Dict[str, t.Any]],
+        batch_options: t.Optional[t.Dict[str, t.Any]],
+        model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
+    ):
+        super().__init__(name, resource_quota, batch_options)
+        model_info, model_file = _get_model_info(tag, model_store)
+        self._model_store = model_store
+        self._model_info = model_info
+        self._model_file = model_file
+        self._function_name = function_name
+
+    @property
+    def num_replica(self) -> int:
+        return 1
```

Comment:
```suggestion
        return max(int(self.resource_quota.cpu), 1)
```

Reply:
unlike other framework runners, most pickable is just a python function that doesn't take advantage of multiple cpu cores, so we will need multiple processes(replicas) here